src/components/Deck/Deck.tsx

```diff
@@ -0,0 +1,12 @@
+
+
+
+const Deck = ({  }) => {	
+	return (
+		<section className="deck">
+
+		</section>
+	);
+};
+
+export default Deck;
```

Comment:
Please add new line at the end of each file. Also, add prettier and eslint to your project